rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -169,6 +169,16 @@ struct CommonMetricFields {
     hr_retention_days: Option<u8>,
     #[serde(skip_serializing_if = "Option::is_none")]
     day_retention_days: Option<u8>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    record_meta: Option<u8>,
+}
+
+fn should_record_meta(use_case_id: String) -> Option<u8> {
+    match use_case_id.as_str() {
```

Comment:
It doesn't seem like you need a String object here. You could do the following here
```suggestion
fn should_record_meta(use_case_id: &str) -> Option<u8> {
    match use_case_id {
```